src/modules/dpos/validator/components/RegisterDelegateForm/InputLabel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Tooltip from 'src/theme/Tooltip';
+import styles from './form.css';
+
+const InputLabel = ({ field }) => {
```

Comment:
as component should be reusable could you pass title and tip from the parent so it can be reuse in the other cases